package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "reservations",
+  "version": "1.0.0",
+  "description": "reservation module for mock up Open Table page",
```

Comment:
@LittleJennie 
@cminho921 

This should not mention Open Table--remember that this organization will stand publicly, and is about demonstrating your ability to build a component that meets a need or has certain functionality vs. your ability to clone something.

Reply:
Thank you for point this out! Description will be updated in the next PR.